content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,744 @@
+  one or two networks
+
+- [Control plane topologies](/docs/ops/deployment/deployment-models#control-plane-models):
+  multi-primary, primary-remote
+
+Before you begin, review the [deployment models guide](/docs/ops/deployment/deployment-models)
+which describes the foundational concepts used throughout this guide.
+
+## Requirements
+
+This guide requires that you have two Kubernetes clusters with any of the
+[supported Kubernetes versions](/docs/setup/platform-setup).
+
+The API Server in each cluster must be accessible to the other clusters in the
+mesh. Many cloud providers make API Servers publicly accessible via network
+load balancers (NLB). If the API Server is not directly accessible, you will
+have to modify the installation procedure to enable access. For example, the
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) gateway used in
+the multi-network and primary-remote configurations below could also be used
+to enable access to the API Server.
+
+## Environment Variables
+
+This guide will refer to two clusters named `CLUSTER1` and `CLUSTER2`. The following
+environment variables will be used throughout to simplify the instructions:
+
+Variable | Description
+-------- | -----------
+`CTX_CLUSTER1` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the `CLUSTER1` cluster.
+`CTX_CLUSTER2` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the `CLUSTER2` cluster.
+
+For example:
+
+{{< text bash >}}
+$ export CTX_CLUSTER1=cluster1
+$ export CTX_CLUSTER2=cluster2
+{{< /text >}}
+
+## Configure Trust
+
+A multicluster service mesh deployment requires that you establish trust
+between all clusters in the mesh. Depending on the requirements for your
+system, there may be multiple options available for establishing trust.
+See [certificate management](/docs/tasks/security/cert-management/) for
+detailed descriptions and instructions for all available options.
+Depending on which option you choose, the installation instructions for
+Istio may change slightly.
+
+This guide will assume that you use a common root to generate intermediate
+certificates for each cluster. Follow the [instructions](/docs/tasks/security/cert-management/plugin-ca-cert/)
+to generate and push a ca certificate secrets to both the `CLUSTER1` and `CLUSTER2`
+clusters.
+
+{{< tip >}}
+If you currently have a single cluster with a self-signed CA (as described
+in [Getting Started](/docs/setup/getting-started/)), you need to
+change the CA using one of the methods described in
+[certificate management](/docs/tasks/security/cert-management/). Changing the
+CA typically requires reinstalling Istio. The installation instructions
+below may have to be altered based on your choice of CA.
+{{< /tip >}}
```

Comment:
Could we have some clarification around `Changing the
CA typically requires reinstalling Istio`?  Do we mean just reinstall the control plane?  Is restarting data plane needed or can Istio auto distribute the new key/certs to each sidecar?

Reply:
cc @myidpt @howardjohn